src/raft.rs

```diff
@@ -1826,6 +1826,7 @@ impl<T: Storage> Raft<T> {
                     request_ctx: m.take_entries()[0].take_data(),
                 };
                 self.read_states.push(rs);
+                self.raft_log.maybe_commit(m.index, m.term);
```

Comment:
Term may be wrong.

Reply:
`index` and `term` in MsgReadIndexResp is the leader's commit index and ite current term, and log entry in the leader's commit index will always have the leader's current term, because the leader only handle MsgReadIndex after it has committed log entry in its term.

Reply:
Get it. Can you also comment it in the file?

Reply:
Remember to also send the patch to etcd.

Reply:
> Get it. Can you also comment it in the file?

Added.


> Remember to also send the patch to etcd.

Will do.